exchanges.yml

```diff
@@ -0,0 +1,396 @@
+  license:
+    name: W3C Software and Document License
+    url: http://www.w3.org/Consortium/Legal/copyright-software.
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+  /workflows:
+    post:
+      summary: Creates a new workflow and returns its information in the response body.
+      tags:
+        - Credentials
+      security:
+        - networkAuth: []
+        - oAuth2: []
+        - zCap: []
+      operationId: createWorkflow
+      description: Creates a new workflow and returns workflowId in the response body.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateWorkflowRequest"
+      responses:
+        "200":
+          description: Workflow successfully created!
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CreateWorkflowResponse"
+        "400":
+          description: Invalid input
+        "401":
+          description: Not Authorized
+        "500":
+          description: Internal Error
+  /workflows/{localWorkflowId}:
+    get:
+      summary: Gets the configuration of an existing workflow and returns it in the response body.
+      tags:
+        - Credentials
+      security:
+        - networkAuth: []
+        - oAuth2: []
+        - zCap: []
+      operationId: getWorkflowConfiguration
+      description: Gets the configuration of an existing workflow and returns it in the response body.
+      responses:
+        "200":
+          description: Workflow configuration retrieved!
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetWorkflowResponse"
+        "400":
+          description: Invalid input
+        "401":
+          description: Not Authorized
+        "500":
+          description: Internal Error
+  /workflows/{localWorkflowId}/exchanges:
+    post:
+      summary: Creates a new exchange and returns exchangeId and time to live in the response body.
+      tags:
+        - Credentials
+      security:
+        - networkAuth: []
+        - oAuth2: []
+        - zCap: []
+      operationId: createExchange
+      description: Creates a new exchange and returns exchangeId and time to live in the response body.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateExchangeRequest"
+      responses:
+        "200":
+          description: Exchange successfully created!
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CreateExchangeResponse"
+        "400":
+          description: Invalid input
+        "401":
+          description: Not Authorized
+        "500":
+          description: Internal Error
+  /workflows/{localWorkflowId}/exchanges/{localExchangeId}:
```

Comment:
`localWorkflowId` and `localExchangeId` must be defined in a `parameters` section below this